### ISSUE TITLE
Extra space when animating dialog from a position other than center or custom

### ISSUE THREAD

**Issue by Dodecadaemon** [fgilde/MudBlazor.Extensions#18]
Hello,

I've been fiddling around with this extension now for a whole day and I'm really liking it so far. I have a product owner who insists on having dialogs fly in from from different sides depending on different scenarios. The dialog needs to fly in from a given side and take up the entire screen until the user closes it.

This extension is almost perfect in meeting that requirement, but there's some extra space that appears on the sides that I can't figure out how to resolve without some JS/CSS hacks. When setting the DialogOptionsEx Position to anything other than custom or center, there's always a 32 pixel space on the top/bottom/left/right depending on what the position is set to. Setting DisableSizeMarginY, DisableSizeMarginX, and DisablePositionMargin aren't having an effect if the animation is moving along that axis.

For example, setting the animation to SlideIn, FullScreen to true, DisableSizeMarginY to true, and Position to BottomCenter leaves a 32 pixel gap at the bottom and pushes the top side of the dialog off screen by the same amount. Alternatively, setting the animation to SlideIn, FullScreen to true, DisableSizeMarginX to true, and Position to CenterRight leaves a 32 pixel gap on the right side pushing content off screen to the left. This happens no matter the animation, and the same gap is present whether fullscreen is true or false. Setting the Position to Center resolves the offsetting issue but then the animations don't move in the desired directions.

I could be missing something, as there's a GIF on the dialog section of the readme that shows a dialog flying out in the way I would expect but with no gap on the side. Here's the function I've been testing with:

```
private void OpenFullScreenDialog<TDialogType>(string title = "") where TDialogType : ComponentBase
{
        var options = new DialogOptionsEx
        {
            MaximizeButton = true,
            CloseButton = true,
            Position = DialogPosition.CenterRight,
            Animations = new AnimationType[] { AnimationType.SlideIn },
            AnimationDuration = TimeSpan.FromMilliseconds(500),
            FullScreen = true,
            DisableSizeMarginX = true,
        };

        _dialogService.ShowEx<TDialogType>(title, options);
}
```

**Comment by Dodecadaemon** [comment]
Thank you for the quick response. I hope you enjoy the rest of your vacation!

**Comment by fgilde** [comment]
So as promised I fixed it with version 1.7.34
Now DisablePositionMargin should work for you
